lib/ansible/modules/system/syspatch.py

```diff
@@ -123,7 +125,6 @@ def syspatch_run(module):
             run_flag = ['-r']
     elif module.params['apply']:
         check_flag = ['-c']
-        run_flag = []
```

Comment:
When `apply` is True, `run_flag` should be `[]` so when `syspatch(8)` is run:

```
        rc, out, err = module.run_command([cmd] + run_flag)
```

it will apply the available patches.

Reply:
Nice catch.